update

krafi-code opened the pull request:
Your code is incredible! It's essentially an open-source game inspired by "Just Shapes and Beats." I've added numerous features, worked tirelessly all day, and here are the enhancements:

I've implemented a tkinter GUI where you can input your YouTube link.
I've integrated presets and added the capability to adjust audio volume with AudioDbAdjuster from the pydub library.
Although the game originally had three levels, I've expanded it to include six exciting levels.
You now have the freedom to customize the colors of the player square, background, and warning elements using tkinter.
I've introduced a mouse mode that you can toggle on or off.
Simply hit the spacebar to make your player jump, and it works like clockwork every 3 seconds.
Players start with four lives, and as their lives decrease, the player color transitions slowly from green to red, creating a captivating animation.
These enhancements make your game even more captivating and engaging!

you can  merge my code

Commit:
update

krafi-code committed:
Update README.md

ChezyName commented:
Great Settings,
small bugs when game restarts color for character sometimes bugged
+ small performance loss
overall great progress.

krafi-code commented:
I was wondering how the program ended up with bugs. I'm sorry for making your code messy; I added too many features and didn't write clean code. The main issue seems to be the writing and reading of the temporary file I added. I did that as a last-HOUR when when I had no more energy left to work 🤣🤣. I did all of this in just 2 days. It would be great if you could make a video about it.How much of my added code have you read? Was it confusing?

krafi-code commented:
Bro, I want your social media account to contract with you, do you have any of telegram, messenger, what's app, Instagram?? Anyone of them ?
https://gitlab.com/krafi/git-backed-diary check out this project maybe you will be interested to check out recently I was working on it.
And my personal blogging website www.krafi.info if you want social media link link.krafi.info ( basically contract button of the main website)
